content/en/docs/components/katib/user-guides/metrics-collector.md

```diff
@@ -73,6 +73,9 @@ To define the pull-based metrics collector for your Experiment:
 
```

Comment:
It looks like that Markdown renders this text incorrectly due to JSON format.
Can you try to remove **```json** from the above line ?

Reply:
Looks fine to me in the deployed preview:

![Captura de pantalla 2025-02-14 a la(s) 16 50 47](https://github.com/user-attachments/assets/1aeb9f0d-349d-4cb0-ac49-19e2cb684869)

Not sure what you're seeing but if it's a pre-existing issue, I think it shouldn't block this

Reply:
Yeah. the preview looks right.
I was trying to say that removing ```json would help us with markdown rendering at the GitHub, but we can address it later.
/lgtm
/approve